### ISSUE TITLE
Unable to figure out CTS tests for Android NNAPI for  SOFTMAX inputs/outputs 

### ISSUE THREAD

**Issue by avaish1** [tensorflow/tensorflow#25003]
Have been using Android P for our internal Product Ramp up  and looked into following code where I am unable to figure out , how below  softmax input  and output could match like what the mathematical formula here ?...any one could help me to understand or any link for documentations around it ?
 http://androidxref.com/9.0.0_r3/xref/frameworks/ml/nn/runtime/test/generated/examples/softmax_float_1.example.cpp 

**Describe the documentation issue**

**We welcome contributions by users. Will you be able to update submit a PR (use the [doc style guide](https://www.tensorflow.org/community/documentation)) to fix the doc Issue?**

**Comment by jvishnuvardhan** [comment]
@avaish1 Please post this kind of support questions at [Stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow). There is a big community to support. Github is mainly for addressing bugs in installation and performance. Thanks!

**Comment by avaish1** [comment]
thanx..would post on stackoverflow...

**Comment by jvishnuvardhan** [comment]
@avaish1 Thank you!
